rasa/utils/tensorflow/temp_keras_modules.py

```diff
@@ -13,13 +13,14 @@
 import numpy as np
 import tensorflow as tf
 
-from tensorflow.python.keras.callbacks import Callback, History
-from tensorflow.python.keras.engine import training
-from tensorflow.python.keras.engine import training_utils
-from tensorflow.python.keras.engine import data_adapter
-from tensorflow.python.keras.utils import version_utils
+from keras.callbacks import Callback, History
+from keras.engine import training, training_utils, data_adapter
+from keras.engine.training import _disallow_inside_tf_function
```

Comment:
Why do we need to import these from `keras` as opposed to `tensorflow.keras`?
This might cause an additional reliance on the `keras` package and I am not sure how in-sync they are with the tensorflow package.

Reply:
The release notes say to remove any references to `tensorflow.python.keras`. See [here](https://github.com/tensorflow/tensorflow/releases/tag/v2.6.0):

> The existing code in tensorflow/python/keras is a staled copy and will be removed in future release (2.7). Please remove any imports to tensorflow.python.keras and replace them with public tf.keras API instead.

Reply:
I could do `tensorflow.keras.callbacks`, but there is no `tensorflow.keras.engine`

Reply:
I'm sorry, i read your comment now

Reply:
> you can directly use _disallow_inside_tf_function from there.

I don't think so, because it's a private method. Unless I am doing sth wrong? But when I try to remove I get
<img width="596" alt="Screenshot 2021-10-01 at 11 14 46" src="https://user-images.githubusercontent.com/45405119/135596039-803432da-92d3-4ae8-9087-f8430fcce1c2.png">

Reply:
If you remove `from keras.engine.training import _disallow_inside_tf_function` then you should do `training._disallow_inside_tf_function` since the import above that imports the `training` module.

Reply:
duhh ofc, idk what I was thinking